api/v1beta1/disruption_webhook_test.go

```diff
@@ -502,7 +529,8 @@ func makeValidNetworkDisruption() *Disruption {
 				IntVal: 1,
 			},
 			Network: &NetworkDisruptionSpec{
-				Drop: 100,
+				Drop:  100,
+				Hosts: []NetworkDisruptionHostSpec{{Port: 80}},
```

Comment:
Why add this? Well my happy case test, where i expect no errors, was giving me errors because we made it to the safety net checking for blast radius. Rather than try to disable that feature in the unit tests here, I thought it was better to just make this spec more valid